server/models/Group.js

```diff
@@ -124,6 +124,15 @@ module.exports = function(Sequelize, DataTypes) {
       get() {
         return `${config.host.website}/${this.get('slug')}`;
       }
+    },
+
+    tags: {
+      type: DataTypes.ARRAY(DataTypes.STRING)
+    },
+
+    supercollective: {
```

Comment:
Also I'm not very familiar with the supercollective idea, but depending on how much investment we want to consider and what you'd like to be able to achieve later (e.g only targeting small structures or going toward bigger NGOs, companies, etc):

From a data structure point of view, `isSuperCollective` isn't something to be persisted but to be based on whether it has child collectives; those parent-child relationships would be the thing to store in DB.

Some potential advantages:
- nest collectives within each other on several levels (to have super-super-collectives etc)
- easily move a collective from a supercollective to another or allow having multiple parent collectives
- more easily perform search operations (e.g "find all members under this supercollective")
- implement more generic security rules
- centralized management, e.g send a tweet to everyone who donated over X € in last month
- in long run potentially easier to integrate within entreprise directories (LDAP or whatnot)

E.g create an `OrgTree` table with columns `parent` and `child`, or even just a single column containing a materialized path (a supercollective with ID 50: `50/`; a collective with ID 60 belonging to that supercollective: `50/60/`). (+ potentially register all of them under a root collective: `1/50/60/`).

Optionally users could also be registered within the OrgTree by replacing `UserGroups`, so the materialized path's leaf node could be a groupId or userid, distinguished through an additional column `nodeType` with e.g values `COLLECTIVE`, `HOST`, `MEMBER` or `BACKER`.

(Unclear to me right now whether same table should mix collectives, users and security rules (`HOST`). It could be a slippery slope where we're tempted to put everything in there?)

Materlialized paths normally also allow for fast search operations, even for structures with many levels, using a proper index and JOIN and LIKE operators.

Some similarities with centralized security management tools (turning them on their head, using transparency): https://www.f-secure.com/en/web/business_global/management-portal